tests/end2end-tests/graphstorm-ec/mgpu_test.sh

```diff
@@ -129,7 +129,7 @@ then
 fi
 
 cnt=$(ls -l /data/gsgnn_ec/prediction/user_rating_movie/ | grep predict | wc -l)
-if test $cnt != $NUM_INFO_TRAINERS
+if test $cnt != $NUM_INFO_TRAINERS * 2
```

Comment:
why do we have this change?

Reply:
If we set --preserve-input True, then we have both old prediction result and new prediction result under /data/gsgnn_ec/prediction/user_rating_movie/